omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1757,7 +1779,7 @@ def _sample_pose_near_object(self, obj, pose_on_obj=None, **kwargs):
                 distance_lo, distance_hi = 0.0, 5.0
                 distance = (th.rand(1) * (distance_hi - distance_lo) + distance_lo).item()
                 yaw_lo, yaw_hi = -math.pi, math.pi
-                yaw = (th.rand(1) * (yaw_hi - yaw_lo) + yaw_lo).item()
+                yaw = th.tensor((th.rand(1) * (yaw_hi - yaw_lo) + yaw_lo).item(), dtype=th.float32)
                 avg_arm_workspace_range = th.mean(self.robot.arm_workspace_range[self.arm])
                 pose_2d = th.tensor(
```

Comment:
This will probably need to change from th.tensor to the equivalent of np.concatenate (is that called th.cat??)